02Lesson/01Project/index.html

```diff
@@ -1,9 +1,37 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title></title>
-</head>
-<body>
+  <head>
+    <title>J&B Coffee Co.</title>
+  </head>
+  <body>
+    <img src="http://restaurantguidelubbock.com/wp-content/uploads/2012/07/01-J-b-coffee-co.jpg"/>
+    <h1>J&B Coffee</h1>
+    <br>
+    <h2>ABOUT US</h2>
+    <p>We love coffee and we love the Lubbock community.
+      Whether you need a study or meeting location,
+      or even a place to throw your next shower -
+      we have the space you need.</p>
+      <br>
+      <br>
+      <h2>MENU</h2>
+      <h3>Signature Drinks</h3>
+        <ul>
+          <li>Lavendar Latte</li>
+          <li>Greek Treat</li>
+          <li>Americano</li>
+        </ul>
+      <h3>Lunch Specials</h3>
+        <ul>
+          <li>Grilled Cheese with Pesto</li>
+          <li>Turkey Club</li>
+          <li>Chicken Salad Sandwich</li>
+        </ul>
+        <br>
+        <br>
+      <h3>CONTACT US</h3>
+      <footer>J&B Coffee Co. (806) 796-1114 | Lubbock, TX | © 2014 J&B Coffee Co.</footer>
```

Comment:
Most of the times it makes sense to put text in a its own HTML element to be able to style it with padding, margin etc. But if you don't need it like in this case, I wouldn't add unnecessary HTML elements.